scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
```

Comment:
thing is it will only populate that help if button is visible on screen. so if you go away from workspace it won't be there, not sure if it's an issue or not, but I think the idea is that this help won't change